File: batches_csharp/batch_02_variable_renaming/compare.cs
using System;

class SumOfSquares {
    static int ComputeQuadraticSum(int[] elements) {
        int accumulator = 0;
        foreach (int value in elements) accumulator += value * value;
        return accumulator;
    }

    static int HandleInput(int[] collection) => ComputeQuadraticSum(collection);

    static void Main() {
        int[] inputData = {1, 2, 3, 4, 5};
        Console.WriteLine(HandleInput(inputData));
    }
}

File: batches_java/batch_02_variable_renaming/compare.java
public class SumOfSquares {
    public static int computeQuadraticSum(int[] elements) {
        int accumulator = 0;
        for (int value : elements) {
            accumulator += value * value;
        }
        return accumulator;
    }

    public static int handleInput(int[] collection) {
        return computeQuadraticSum(collection);
    }

    public static void main(String[] args) {
        int[] inputData = {1, 2, 3, 4, 5};
        System.out.println(handleInput(inputData));
    }
}
